internal/taskfile/version/version.go

```diff
@@ -10,6 +10,8 @@ var (
 	v21 = mustVersion("2.1")
 	v22 = mustVersion("2.2")
 	v23 = mustVersion("2.3")
+	v24 = mustVersion("2.4")
+	v25 = mustVersion("2.5")
```

Comment:
@andreynering. please correct me if I am wrong.

2.4 and 2.5 have been released, so when adding a new feature to the taskfile here, it should probably come as a 2.6 version.

Since there are not checks for 2.4 and 2.5 today, I presume these two releases does not provide any updated to the Taskfile format, and does not need the `v2X` and `Isv2X()` definitions?